packages/app-builder/src/routes/_builder+/settings+/inboxes.$inboxId.tsx

```diff
@@ -231,24 +232,14 @@ export default function Inbox() {
               header: t('settings:inboxes.inbox_details.auto_assign_enabled.label'),
               size: 150,
               cell: ({ getValue, row }) => {
-                const fetcher = useFetcher<typeof UpdateInboxUser>();
                 const [value, setValue] = useState(getValue());
+                const { mutate: editAutoAssign } = useEditAutoAssignMutation();
```

Comment:
I would prefer if we use `const editAutoAssignMutation = useEditAussignMutation()` rather than spreading the object.
It makes it much more clear when you have multiple queries/mutations and to have a consistency between the uses of queries/mutations I think it's better to do it even if we have only one.

What do you think?